source/diagnostics.tex

```diff
@@ -482,7 +482,7 @@
 If the evaluation yields \tcode{true} there are no further effects.
 \item
 Otherwise, the \tcode{assert} macro's expression
-creates a diagnostic on the standard error stream in an
+creates a diagnostic on the standard error stream (\IsoC{}, 7.23.3) in an
```

Comment:
The `\xrefc` macro will add "SEE ALSO:" and thus it seems unsuitable to me to use it here.